sysbox-in-docker/README.md

```diff
@@ -0,0 +1,91 @@
+# Sysbox-In-Docker
+
+As its name implies, Sysbox-In-Docker aims to provide a containerized environment
+where to execute the Sysbox runtime.
+
+The goal is to offer a simple and non-invasive environment where users can quickly
+interact with the Sysbox runtime. Note that, for the general use-case, Sysbox is
+expected to operate in a regular (non-containerized) environment (i.e., host
+installation).
+
+## Requirements
+
+There are a few requirements that must be satisfied for Sysbox to properly operate
+in a containerized environment:
+
+* Supported Distros: As it's the case for the regular host-based installation,
+Sysbox is currently only supported in the distributions shown in this
+[doc](../docs/distro-compat.md).
+
+* Root-privileges: As a container runtime, Sysbox requires root privileges to
+operate. As a result, Sysbox-In-Docker container must be launched in "privileged"
+mode.
+
+    **Note**: Within the privileged container, inner containers launched with Docker +
+    Sysbox will be strongly isolated from the host by Sysbox (e.g., via the Linux
+    user-namespace).
+
+* Required bind-mounts:
```

Comment:
More detail here:

In order for the sysbox-in-docker container to operate properly, the following host bind-mounts are required:

* /some/host/dir/ -> /var/lib/docker
* /another/host/dir -> /var/lib/sysbox
* /lib/modules/<kernel> -> /lib/modules/<kernel>
* /usr/src/linux-headers-<kernel> -> /usr/src/linux-headers-<kernel>

Reply:
done